osu.Game/Screens/Play/HUD/ClicksPerSecond/ClicksBpmCounter.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Localisation;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+
+namespace osu.Game.Screens.Play.HUD.ClicksPerSecond
+{
+    public partial class ClicksBpmCounter : ClicksPerSecondCounter
+    {
+        [Resolved]
+        private ClicksPerSecondController controller { get; set; } = null!;
+
+        protected override double RollingDuration => 100;
+
+        protected override void Update()
+        {
+            base.Update();
+
+            // multiply by 60 * (1 / 4) to convert CPS to BPM
+            Current.Value = (int)(controller.Value * 60f * (1f / 4f));
```

Comment:
I dunno. At the point where *this* is the extent of the implementation I'd probably rather have this as a setting on the clicks per second counter (maybe name it something like "unit" or something). Feels like much ado about nothing otherwise.

Reply:
> I dunno. At the point where _this_ is the extent of the implementation I'd probably rather have this as a setting on the clicks per second counter (maybe name it something like "unit" or something). Feels like much ado about nothing otherwise.

The change in implemented, the bpm counter is one of the unit options of `ClickSpeedCounter`